_acs-aem-commons/features/packagers/asset-packager/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: acs-aem-commons_feature
+title: Asset Packager
+description: Easily package pages and the associated assets
+redirect_from: /acs-aem-commons/features/asset-packager.html
+date: 2019-05-20
+feature-tags: administration
+initial-release: 4.2.0
+---
+
+# Purpose
+
+The Asset Packager lets you create a content package that is given a specific page path and will iterate over the content of that package to add all referenced assets into the package as well, giving you exactly what you need for a single page with all images.
```

Comment:
The part about "single page with all images" is a bit misleading b/c by default the packager will pull a page and all of its descendants, requiring the user to configure a regex to avoid.  I feel like this should instead talk to the default behaviour, which is a "page tree", though it can mention that using exclusions can limit it to a single page.

Reply:
Updated in the latest commits